### ISSUE TITLE
Add eu-north-1 to region explorer

### ISSUE THREAD

**Issue by gitcatrat** [aws/aws-toolkit-vscode#1291]
**Is your feature request related to a problem? Please describe.**
...

**Describe the solution you'd like**
`eu-north-1` region can be added to Explorer.

**Describe alternatives you've considered**
...

**Additional context**
...

**Comment by bryceitoc9** [comment]
Hi @gitcatrat ,

I can definitely add `eu-north-1` to my explorer:

![image](https://user-images.githubusercontent.com/29374703/92959472-ba4d9980-f420-11ea-8800-4be3caf75892.png)

Can you send us a list of what shows up when you try to add a region, as well as a list of what shows up when you try to remove a region? Additionally, can you send us the version of the toolkit that you're using? Please note that if you're using credentials for CN or GovCloud, you'll only see those regions show up in the selection box (and vice versa, you won't see those regions if you're using non-partitioned credentials).

**Comment by gitcatrat** [comment]
@bryceitoc9  Ahh, after I toggled AWS: EXPLORER, I can see both `eu-west-1` and `eu-north-1`. I guess they were added automatically based on cedentials? Here's a full list of `eu` regions I was seeing before this discovery, presumably missing already added regions.

![eu-regions](https://user-images.githubusercontent.com/58787197/92974828-c02e9380-f48f-11ea-86c7-d63eb12885a8.png)

Sorry for spamming and thank you!
